Makefile

```diff
@@ -56,8 +56,8 @@ else
 	ifeq ($(VERSION), )
 		VERSION = 3.2.0
 	endif
-	# Specifies the namespace where the operator pods are deployed (defaults to opendatahub-operator-system)
-	OPERATOR_NAMESPACE ?= opendatahub-operator-system
+	# Specifies the namespace where the operator pods are deployed (defaults to redhat-ods-operator)
+	OPERATOR_NAMESPACE ?= redhat-ods-operator
```

Comment:
do we want to change this?
if we do e2e test for rhoai, then it uses the real/production namespace ?

Reply:
Reverted the Makefile changes